helm_deploy/values-preprod.yaml

```diff
@@ -9,7 +9,9 @@ generic-service:
   env:
     APPLICATIONINSIGHTS_CONFIGURATION_FILE: applicationinsights.json
     OAUTH_ENDPOINT_URL: "https://sign-in-preprod.hmpps.service.justice.gov.uk/auth"
+    PRISON_API_URL: "https://api-preprod.prison.service.justice.gov.uk"
     BASM_ENDPOINT_URL: "https://hmpps-book-secure-move-api-preprod.apps.live-1.cloud-platform.service.justice.gov.uk"
+    PRISONER_SEARCH_API_ENDPOINT_URL: "https://prisoner-offender-search-preprod.prison.service.justice.gov.uk"
```

Comment:
Would be nice to make this a bit more consistent and remove the API and just use ENDPOINT?

PRISON_ENDPOINT_URL 
PRISONER_SEARCH_ENDPOINT_URL